src/utils/identity.js

```diff
@@ -0,0 +1,3 @@
+export default function identity(value) {
+  return value;
+}
```

Comment:
> It's probably best to put these all in a single file and export them individually

:+1: 

> using an arrow function would be better

I think it's just a matter whether we want to have it as a named function or not.